public/pages/DetectorsList/List/List.tsx

```diff
@@ -12,90 +12,132 @@
 import { AppState } from '../../../redux/reducers';
 import { getDetectorList } from '../../../redux/reducers/ad';
-import { APP_PATH, BREADCRUMBS, PLUGIN_NAME } from '../../../utils/constants';
+import {
+  getIndices,
+  getPrioritizedIndices,
+} from '../../../redux/reducers/elasticsearch';
+import { APP_PATH, PLUGIN_NAME } from '../../../utils/constants';
+import { getVisibleOptions, sanitizeSearchText } from '../../utils/helpers';
 import { EmptyDetectorMessage } from '../Components/EmptyMessage/EmptyMessage';
 import { ListControls } from '../Components/ListControls/ListControls';
+import {
+  MAX_DETECTORS,
+  MAX_SELECTED_INDICES,
+  GET_ALL_DETECTORS_QUERY_PARAMS,
+  ALL_DETECTOR_STATES,
+  ALL_INDICES,
+  DETECTOR_STATE,
+} from '../../utils/constants';
 import { getURLQueryParams } from '../utils/helpers';
+import {
+  filterAndSortDetectors,
+  getDetectorsToDisplay,
+} from '../../utils/helpers';
 import { staticColumn } from '../utils/tableUtils';
 
 export interface ListRouterParams {
   from: string;
   size: string;
   search: string;
+  indices: string;
   sortDirection: SORT_DIRECTION;
   sortField: string;
 }
 interface ListProps extends RouteComponentProps<ListRouterParams> {}
 interface ListState {
   page: number;
   queryParams: GetDetectorsQueryParams;
+  selectedDetectorStates: DETECTOR_STATE[];
+  selectedIndices: string[];
 }
-const MAX_DETECTORS = 1000;
 
 export const DetectorList = (props: ListProps) => {
   const dispatch = useDispatch();
-  const detectors = useSelector((state: AppState) => state.ad.detectorList);
-  const totalDetectors = useSelector(
-    (state: AppState) => state.ad.totalDetectors
+  const allDetectors = useSelector((state: AppState) => state.ad.detectorList);
+  const elasticsearchState = useSelector(
+    (state: AppState) => state.elasticsearch
   );
+
   const isLoading = useSelector((state: AppState) => state.ad.requesting);
 
+  // Getting all initial indices
+  const [indexQuery, setIndexQuery] = useState('');
+  useEffect(() => {
+    const getInitialIndices = async () => {
+      await dispatch(getIndices(indexQuery));
+    };
+    getInitialIndices();
+  }, []);
+
+  // Updating displayed indices (initializing to first 20 for now)
+  const visibleIndices = get(elasticsearchState, 'indices', []) as CatIndex[];
+  const visibleAliases = get(elasticsearchState, 'aliases', []) as IndexAlias[];
+  const indexOptions = getVisibleOptions(visibleIndices, visibleAliases);
+
   const [state, setState] = useState<ListState>({
     page: 0,
     queryParams: getURLQueryParams(props.location),
+    selectedDetectorStates: ALL_DETECTOR_STATES,
+    selectedIndices: ALL_INDICES,
   });
 
-  // Set Breadcrumbs and initial queryParams
+  // Remove breadcrumbs on page initialization
   useEffect(() => {
-    chrome.breadcrumbs.set([
-      BREADCRUMBS.ANOMALY_DETECTOR,
-      BREADCRUMBS.DASHBOARD,
-    ]);
+    chrome.breadcrumbs.set(['']);
   }, []);
```

Comment:
QQ: Why are we removing this , is it because List is root page?

Reply:
Correct, Lovinda wants no breadcrumbs on this page nor the dashboard page.

Reply:
Should we at least have Root ? Just Anomaly Detector ?

Reply:
She wants nothing there, see https://aws-uxdr.invisionapp.com/d/?redirHash=#/console/19392952/405140830/preview?scrollOffset=4200

Reply:
Cool. I don't like that but I'd let her decide.